src/main/java/kafdrop/service/KafkaMonitorImpl.java

```diff
@@ -116,27 +116,31 @@ public ClusterSummaryVO getClusterSummary(Collection<TopicVO> topics) {
   }
 
   private Map<String, TopicVO> getTopicMetadata(String... topics) {
-    final var topicInfos = highLevelConsumer.getTopicInfos(topics);
-    final var retrievedTopicNames = topicInfos.keySet();
-    final var topicConfigs = highLevelAdminClient.describeTopicConfigs(retrievedTopicNames);
-
-    for (var topicVo : topicInfos.values()) {
-      final var config = topicConfigs.get(topicVo.getName());
-      if (config != null) {
-        final var configMap = new TreeMap<String, String>();
-        for (var configEntry : config.entries()) {
-          if (configEntry.source() != ConfigSource.DEFAULT_CONFIG &&
-              configEntry.source() != ConfigSource.STATIC_BROKER_CONFIG) {
-            configMap.put(configEntry.name(), configEntry.value());
-          }
-        }
-        topicVo.setConfig(configMap);
-      } else {
-        LOG.warn("Missing config for topic {}", topicVo.getName());
-      }
-    }
-    return topicInfos;
-  }
+		final var topicInfos = highLevelConsumer.getTopicInfos(topics);
+		final var retrievedTopicNames = topicInfos.keySet();
+		try {
+			final var topicConfigs = highLevelAdminClient.describeTopicConfigs(retrievedTopicNames);
+			for (var topicVo : topicInfos.values()) {
+				final var config = topicConfigs.get(topicVo.getName());
+				if (config != null) {
+					final var configMap = new TreeMap<String, String>();
+					for (var configEntry : config.entries()) {
+						if (configEntry.source() != ConfigSource.DEFAULT_CONFIG
+								&& configEntry.source() != ConfigSource.STATIC_BROKER_CONFIG) {
+							configMap.put(configEntry.name(), configEntry.value());
+						}
+					}
+					topicVo.setConfig(configMap);
+				} else {
+					LOG.warn("Missing config for topic {}", topicVo.getName());
+				}
+			}
+		} catch (KafkaAdminClientException exc) {
```

Comment:
I'm not sure this is the right approach. We can't just suppress a `KafkaAdminClientException` because it could be a legitimate failure for a different reason.

You mention this fails with an authorization error. Looking at the code, it should work if the admin client has read-only access. Where do you suppose this requires write access?